### PR TITLE
Created April's Clicky Reflection

### PR DESCRIPTION
**Pull request by alm7468**
This is my clicky post reflection. It could use a proof read to make sure everything looks okay.

### CODE REVIEW

File: _posts/2022-10-04-clicky-reflection-alm7468.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+author: alm7468
+title: “April’s Clicky Reflection”
+---
+
+I’m trying to do a simple I-Spy point-and-click game experience. 
+I created the intro setup where 4 options are presented. The user is prompted to double click one option (double clicking is a way to incorporate the required trinket screen click + user selection click). The screen should clear, the puzzle image becomes the background, and the user is prompted to click on a particular item.
```
Comment (amptex101) [inline _posts/2022-10-04-clicky-reflection-alm7468.md:8]: This is a really cool interface/concept, and I am also wondering how to only keep the puzzle image in the background and get rid of the 4 rectangles once you enter the game

**Review by amptex101 — COMMENTED** [review]
Looks good to me!

**Review by amptex101 — APPROVED** [review]
Approved!

File: _posts/2022-10-04-clicky-reflection-alm7468.md (hunk 1 above)
Comment (alm7468) [reply _posts/2022-10-04-clicky-reflection-alm7468.md:8]: Yea I am still working on it! But this is where I am at right now.

File: _posts/2022-10-04-clicky-reflection-alm7468.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (eah13) [inline _posts/2022-10-04-clicky-reflection-alm7468.md:1]: This file should be in the `alm7468` subdirectory. So I believe the overall filename would be:

~~~
_posts/alm7468/2022-10-04-clicky-reflection-alm7468.md
~~~

File: _posts/2022-10-04-clicky-reflection-alm7468.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+layout: post
+author: alm7468
+title: “April’s Clicky Reflection”
```
Comment (eah13) [inline _posts/2022-10-04-clicky-reflection-alm7468.md:4]: These `"` might be smartquotes. Did you paste from a word processing program?

I'm not sure it'll be a problem, but if there are any problems, this is a place to look.

**Review by eah13 — DISMISSED** [review]
Looks good overall! 

I think this post isn't in the right subfolder of `_posts`.  Fix the subfolder (via an edit to filename) and it's good to go.

File: _posts/2022-10-04-clicky-reflection-alm7468.md (hunk 3 above)
Comment (alm7468) [reply _posts/2022-10-04-clicky-reflection-alm7468.md]: I swapped the quotes out, thank you!

File: _posts/2022-10-04-clicky-reflection-alm7468.md (hunk 2 above)
Comment (alm7468) [reply _posts/2022-10-04-clicky-reflection-alm7468.md]: I added the subfolder, thank you!

**Pull request merged**